### ISSUE TITLE
remaining work with PR #1923 

### ISSUE THREAD

**Issue by junwang-noaa** [ufs-community/ufs-weather-model#1962]
## Description
Following work needs to be done after  PR#1923.
 
- merge two namelist templates: cpld_control_ugwpv1.nml.IN and cpld_control.nml.IN.
- enable the cpld_control_gfsv17 related tests on gaea and jet
- on fv3 side, merge the ccpp suite files: suite_FV3_GFS_v17_coupled_p8_ugwpv1.xml and suite_FV3_GFS_v17_coupled_p8.xml.
- merge control_p8 and control_p8_ugwpv1 into one test, such as control_global

This will allow further updates in the GFSv17/GEFSV13 regression test.

**Comment by JessicaMeixner-NOAA** [comment]
To enable the cpled_control_gfs and related tests on gaea and jet the input from hera in /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/input-data-20221101/WW3_input_data_20220624 needs to be sync'd to the input locations.  I confirmed that the required inputs are not on gaea, but I do not have access to jet to know if it's there however I suspect it is not there either.    @junwang-noaa who should I ask about getting the data transferred to gaea and jet?

**Comment by junwang-noaa** [comment]
@jkbk2004 Can your group help moving the data to gaea and jet? It will stay under RT baseline input data directories as shown above. Thanks

**Comment by jkbk2004** [comment]
@JessicaMeixner-NOAA WW3_input_data_20220624 is the one to rsync to gaea and jet, right?

**Comment by JessicaMeixner-NOAA** [comment]
Thanks @junwang-noaa 

@jkbk2004  yes - there's just  a few additional files in that directory missing on gaea (and likely jet).

**Comment by jkbk2004** [comment]
@JessicaMeixner-NOAA can you check gaea: /lustre/f2/pdata/ncep/role.epic/RT/NEMSfv3gfs and jet: /mnt/lfs4/HFIP/hfv3gfs/role.epic/RT/NEMSfv3gfs ? Note that we moved gaea and jet DISKNM.

**Comment by JessicaMeixner-NOAA** [comment]
@jkbk2004 Gaea looks good now.  I cannot check jet as I do not have access.

**Comment by JessicaMeixner-NOAA** [comment]
I tried to run a test on gaea but was unable to get anything to run (not just the tests I tried to enable).

**Comment by jkbk2004** [comment]
@JessicaMeixner-NOAA permission issue? Can you point me to the experiment path?

**Comment by JessicaMeixner-NOAA** [comment]
@jkbk2004 I was on a c5 node (haven't used gaea in a while), I went back and logged in choosing a c4 node and things just started running for me.    

@junwang-noaa  @DeniseWorthen I'm testing all pdlib tests on gaea right now, when I make a PR to run the cpld_control_gfsv17 tests on everything but noaacloud, should I also update the other pdlib intel tests to match or leave as is?

**Comment by DeniseWorthen** [comment]
Yes, I think so. The only reason they were not enabled to start with was the requirement of the scotch lib. If that is now available, there is no reason I know to restrict the platforms.

**Comment by gspetro-NOAA** [comment]
@junwang-noaa Has this issue been resolved? It seems like at least several elements of it were resolved, but I want to be sure before closing.

**Comment by junwang-noaa** [comment]
The code wacommitted, close the issue.
